deploy_nva_api.yml

```diff
@@ -211,7 +211,7 @@ Resources:
         SemanticVersion: 0.1.1
       Parameters:
         CustomDomain: !Ref ApiCustomDomain
-        UserServiceSecretArn: !Sub 'arn:aws:secretsmanager:eu-west-1:${AWS::AccountId}:secret:UserCatalogueApiKey-dIqvz5'
+        UserServiceSecretArn: !Sub 'arn:aws:secretsmanager:eu-west-1:${AWS::AccountId}:secret:UserCatalogueApiKey-p9ebkD'
```

Comment:
It just occurred to me that this ARN must be a template parameter because we have locked the template to one specific envirionment (where the ARN exists) with this change.

Reply:
@svenngjeto @janbj : Could anyone assist me on a videocall and we fix it to use the concept of `{{resolve:secretsmanager:UserCatalogueApiKey:SecretString}}` ? Poke me at slack.

Reply:
We have updated this @janbj , see new commits.